frontend/utils/api.js

```diff
@@ -18,6 +18,29 @@ const formatGroceryItemAdd = (formData) => {
     }
 }
 
+
+
+export const addPrice = (formData) => {
+    const ROUTE = '/price';
+    console.log(formData);
+    return fetch(BACKEND_URL + ROUTE, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify(formData)
+    })
+        .then(response => response.json())
```

Comment:
These lines can probably be integrated into one single 'then'. 
`.then(response => {
        let json = response.json();
        console.log(json);
})`
Also, the resolve function only logs the result without doing anything about it. Does the function actually implemented the user story?

Reply:
Yes, that's something that makes sense